hack/istio/install-testing-demos.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+##############################################################################
+# install-testing-demos.sh
+#
+# Installs all the demo applications needed for cypress testing.
+# Works on both openshift and non-openshift environments.
+##############################################################################
+
+set -e
+
+SCRIPT_DIR="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)"
+
+: ${CLIENT_EXE:=oc}
+
+while [ $# -gt 0 ]; do
+  key="$1"
+  case $key in
+    -c|--client)
+      CLIENT_EXE="$2"
+      shift;shift
+      ;;
+    -h|--help)
+      cat <<HELPMSG
+Valid command line arguments:
+  -c|--client: either 'oc' or 'kubectl'
+  -h|--help: this text
+HELPMSG
+      exit 1
+      ;;
+    *)
+      echo "Unknown argument [$key]. Aborting."
+      exit 1
+      ;;
+  esac
+done
+
+IS_OPENSHIFT="false"
+if [[ "${CLIENT_EXE}" = *"oc" ]]; then
+  IS_OPENSHIFT="true"
+fi
+
+echo "CLIENT_EXE=${CLIENT_EXE}"
+echo "IS_OPENSHIFT=${IS_OPENSHIFT}"
+
+# Installed demos should be the exact same for both environments.
+# Only the args passed to the scripts differ from each other.
+if [[ "${IS_OPENSHIFT}" = "true" ]]; then
+  echo "Deploying bookinfo demo..."
+  /bin/bash "${SCRIPT_DIR}"/install-bookinfo-demo.sh
```

Comment:
a) do you need `/bin/bash` ? The .sh files have the `#!/bin/bash` headers that tell the OS what shell to use.
b) the quotes around `"${SCRIPT_DIR}"` I assume are to protect against spaces in the path? If so, the placing of the quotes looks weird to me. I would think it should just be `"${SCRIPT_DIR}/install-bookinfo-demo.sh"` (the entire string wrapped)